### PR TITLE
Store game messages + translations in database

### PR DESCRIPTION
**Pull request by Brainicism**

### CODE REVIEW

File: src/scripts/game-message-converter.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+const enFile = JSON.parse(
+    fs.readFileSync(path.resolve(__dirname, "../../i18n/en.json")).toString()
+);
+
+const koFile = JSON.parse(
+    fs.readFileSync(path.resolve(__dirname, "../../i18n/ko.json")).toString()
+);
+
+(async () => {
+    if (require.main === module) {
```
Comment (taahamahdi) [inline src/scripts/game-message-converter.ts:18]: Early return instead

File: src/migrations/20220514235230_game-messages-varchar-to-text.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    });
+};
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.down = function (knex) {
+    return knex.schema.table("game_messages", function (table) {
+        table.dropColumn("id");
```
Comment (taahamahdi) [inline src/migrations/20220514235230_game-messages-varchar-to-text.js:26]: When is this column added

**Review by taahamahdi — APPROVED** [review]

File: src/migrations/20220514235230_game-messages-varchar-to-text.js (hunk 2 above)
Comment (Brainicism) [reply src/migrations/20220514235230_game-messages-varchar-to-text.js:26]: `table.increments();` creates a auto-increment primary key called 'id'

File: src/scripts/game-message-converter.ts (hunk 1 above)
Comment (Brainicism) [reply src/scripts/game-message-converter.ts:18]: This is how we do it everywhere

**Pull request merged**